src/Sulu/Bundle/SecurityBundle/Command/CreateRoleCommand.php

```diff
@@ -43,13 +43,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $doctrine = $this->getContainer()->get('doctrine');
         $em = $doctrine->getManager();
-        $now = new \Datetime();
+        $name = $input->getArgument('name');
+        $system = $input->getArgument('system');
+
+        $repository = $em->getRepository('SuluSecurityBundle:Role');
+
+        $role = $repository->findOneByName($name);
+
+        if ($role) {
```

Comment:
No its not part of the interactive part. It is only when you do this non-interactively.